body.tex

```diff
@@ -180,29 +181,28 @@ \section{Long Term Planning}
-\footnote{Note that the level of the milestone is logically distinct from the level of the WBS and OBS, even though the same term (``level'') is used to label both. In the OBS, level 0 is the Observatory, level 1 is the departments, level 2 is the teams, and level 3 is the groups within the teams. We take care to always disambiguate ``WBS level'' from ``milestone level'' and never use the term ``level'' without these qualifiers preceding it.}
+Milestones are allocated to one of four levels, defined as follows:
 
 \begin{description}
-%this is not consistent with NOIRLab or our current FY21 plan. We need to discuss an decide how to capture these. Could be one or two RDO level MS and several L2 MS for each department both of which show up in the POP. Or we keep the Major MS classification and have several of those in each Dep. Suppose we had one or two L1 milestones like Annual DR. Then all the current Major MS in our sheet could be L2s that point at that.
 \item[Level 1]
-These are at the full observatory level owned by the directors office
+These are at the full observatory level and owned by the Directors Office. 
+An example is the completion of an annual Data Release.
 \item[Level 2]
-These reflect cross department commitments. As such, they must be defined
-in consultation with the Director's Office
+These reflect effort within a Department and are owned by an Associate Director, or are cross-Department commitments. 
+As such, they must be defined in consultation with the Director's Office.
 \item[Level 3]
-These are internal to a particular Department and assigned to a team  and can therefore be specified by a single team lead.
+These are internal to a particular Department and assigned to a team and can therefore be specified by a single team lead.
 \end{description}
 
-Some of these milestones are exposed to external reviewers: it is vital that these be delivered on time and to specification.
+Some of these milestones are exposed to external reviewers (usually reserved for Level 1 milestones): it is vital that these be delivered on time and to specification.
 Low-level (3 and beyond) milestones are defined for use within Departments, but even here properly adhering to the plan is vital: your colleagues in other teams will use these milestones to align their schedules with yours, so they rely on you to be accurate.
 
 Epics should help achieve milestones i.e. they may be blocking issues on the milestones.
-A detailed description of work for a given \gls{epic} is known, it can and should be described in \gls{JIRA}.
-This should be assigned to the appropriate \glspl{cycle}.
+When a detailed description of work for a given \gls{epic} is known, it is described in \gls{JIRA}.
+It should then be assigned to the appropriate \glspl{cycle}.
```

Comment:
We could add a note here about how each staff member's work plan is laid out in the form of the set of epics that they own. One of their goals for the year is then to complete those epics. I think this approach will ensure that this goal is "SMART" and hence effective to track via their performance evaluations. When writing it down, the goal should still include a one sentence summary of what those epics will collectively achieve. Can we look at an aggregated summary of an example individual's epics' metadata? Wondering if we can provide this to help us and the staff visualize what they are supposed to be doing for the year.